src/test/java/io/stargate/sgv2/jsonapi/service/embedding/operation/PropertyBasedOverrideProfile.java

```diff
@@ -13,14 +13,14 @@ public boolean disableGlobalTestResources() {
   @Override
   public Map<String, String> getConfigOverrides() {
     return ImmutableMap.<String, String>builder()
-        .put("jsonapi.embedding.config.store", "property")
-        .put("jsonapi.embedding.provider.openai.enabled", "true")
-        .put("jsonapi.embedding.provider.openai.api-key", "openai-api-key")
-        .put("jsonapi.embedding.provider.openai.url", "https://api.openai.com/v1/")
-        .put("jsonapi.embedding.provider.huggingface.enabled", "true")
-        .put("jsonapi.embedding.provider.huggingface.api-key", "hf-api-key")
-        .put("jsonapi.embedding.provider.huggingface.url", "https://api-inference.huggingface.co")
-        .put("jsonapi.embedding.provider.vertexai.enabled", "false")
+        //        .put("jsonapi.embedding.config.store", "property")
+        .put("jsonapi.embedding.providers.openai.enabled", "true")
```

Comment:
What is the use of this config `.put("jsonapi.embedding.config.store", "property")`? I changed the config mapping so it will have mapping errors, but it seems no effect if I remove it. @maheshrajamani

Reply:
We no longer support `in-memory` options, we can remove it.